README.md

```diff
@@ -4,33 +4,38 @@
 
 Training n°1 for NFT marketplace
 
-Introduction : Business objects managed by a blockchain are called `assets`. On Tezos you will find the term `Financial Asset or FA` with different version 1, 2 or 2.1.
+# Introduction
+
+Business objects managed by a blockchain are called `assets`. On Tezos you will find the term `Financial Asset or FA` with different version 1, 2 or 2.1.
 Here below different categorization of assets.
 
 ![](http://jingculturecommerce.com/wp-content/uploads/2021/03/nft-assets-1024x614.jpg)
 
 # :wine_glass: Wine marketplace
 
 We are going to build a Wine marketplace.
```

Comment:
Please add details!
What features will be available on this marketplace?

A quick list like:

> You are going to build a wine marketplace:
>- add new wine bottles
>- manage your personal wine collection
>- sell some of your bottles
>- buy bottle(s) of other people

Reply:
You can also say that we will focus on the contract code, and so, you will provide the related front-end code to copy/paste.